src/covid_model_seiir_pipeline/forecasting/model/forecast_metrics.py

```diff
@@ -53,16 +54,16 @@ def splice_components(components_past: pd.DataFrame, components_forecast: pd.Dat
 
 
 def get_observed_infecs_and_deaths(infection_data: pd.DataFrame):
-    observed = infection_data['obs_infecs'] == 1
+    observed = infection_data['observed_infections'] == 1
     observed_infections = (infection_data
-                           .loc[observed, ['location_id', 'date', 'cases_draw']]
+                           .loc[observed, ['location_id', 'date', 'infections_draw']]
                            .set_index(['location_id', 'date'])
                            .sort_index()
-                           .rename(columns={'cases_draw': 'infections'}))
-    observed = infection_data['obs_deaths'] == 1
+                           .rename(columns={'infections_draw': 'infections'}))
+    observed = infection_data['observed_deaths'] == 1
     observed_deaths = (infection_data
-                       .loc[observed, ['location_id', 'date', 'deaths_mean']]
-                       .rename(columns={'deaths_mean': 'deaths'})
+                       .loc[observed, ['location_id', 'date', 'deaths_draw']]
+                       .rename(columns={'deaths_draw': 'deaths'})
```

Comment:
What caused the schema/column names to change? Is this stable/the new normal?

Reply:
The input source changed.  But this is a mistake.  The new inputs don't contain the necessary data.  I'm resolving with Ryan.